cmd/nvidia-container-runtime-hook/main.go

```diff
@@ -114,9 +114,19 @@ func doPrestart() {
 	}
 	args = append(args, "configure")
 
-	if !hook.Features.AllowCUDACompatLibsFromContainer.IsEnabled() {
-		args = append(args, "--no-cntlibs")
+	switch hook.NVIDIAContainerRuntimeConfig.Modes.Legacy.CUDACompatMode {
+	case config.CUDACompatModeLdconfig:
+		args = append(args, "--cuda-compat-mode=ldconfig")
```

Comment:
Instead of repeating `args = append(args, "<flag>")` statements, can we assign the resolved flag to a local variable and append it to `args` after the switch-case block?